src/Data/Array/Accelerate/Linear/Vector.hs

```diff
@@ -174,8 +178,8 @@ negated = lift1 (L.negated :: f (Exp a) -> f (Exp a))
 
 -- | Subtraction with a scalar on the right
 --
--- >>> V2 1 2 ^- 3
--- V2 (-2) (-1)
+-- >>> lift (V2 1 2 :: V2 Int) ^- 3
+-- (2,1)
```

Comment:
This part I'm worried about, since the answer that `doctest` gave `(2,1)` appears wrong...

In fact, there are other updated answers in this module that differ from what was originally given. It'd be nice to have someone sanity-check that the answers are in fact correct.